### DISCUSSION TITLE
public folder ignored during build

### DISCUSSION THREAD

**Discussion by iammrsea** [adonisjs/core#1390] [Help] [answered]
I just bootstrapped a new project for V5, and each time I start the server, the public folder gets ignored by typescript. This is very weird, because my previous project with V5 was copying the public folder over the build directory. Any ideas what could be the reason for this?

**Comment by iammrsea** [comment, marked as answer]
Never mind, I forgot to add static.ts file and register 'public/**' in .adonisrc.json file

**Reply by ahmed19992104** [reply]
can you send me text of static.ts because it is not exists

**Reply by thetutlage** [reply]
Here's the file https://github.com/adonisjs/core/blob/develop/templates/config/static.txt
